src/test/resources/regressions/features/loops/loops-continue-label1.gobra

```diff
@@ -0,0 +1,14 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+func foo() {
+l:
+	invariant true
+	for true {
+		for true {
+			continue l
+		}
+	}
```

Comment:
please, add some assertions. It can be the easiest assertion that you can think of that is not true.